workspaces/orchestrator/app-config.yaml

```diff
@@ -81,7 +81,7 @@ auth:
 # permission:
 #   enabled: true
 #   rbac:
-#     policies-csv-file: ../../plugins/orchestrator/docs/rbac-policy.csv
```

Comment:
If relative link is used then it is from `rhdh-plugins/workspaces/orchestrator/packages/backend`